### PR TITLE
feat(fe-fpm-writer): apply context path to a building block based on min ui5 version

### PR DESCRIPTION
**Pull request by AlinaGovoruhina**
Apply contextPath for minimum UI5 versions equal or below 1.96 ( + always for chart building block).
Check for minimum UI5 version when applying the building block and when getting the code snippet content.

### CODE REVIEW

File: packages/fe-fpm-writer/test/unit/prompts/prompts.test.ts [hunk 1]
```diff
@@ -246,6 +247,20 @@ describe('Prompts', () => {
             expect(result.viewOrFragmentPath.filePathProps?.fileName).toBe('Main.view.xml');
         });
 
+        test.each(types)('Type "%s", get code snippet, min ui5Version = 1.95.0', async (type: PromptsType) => {
```
Comment (815are) [inline packages/fe-fpm-writer/test/unit/prompts/prompts.test.ts:250]: could you please add test to version like `1.96.34` or there is already such test?

File: packages/fe-fpm-writer/src/building-block/index.ts [hunk 2]
```diff
@@ -156,25 +172,30 @@ function getMetaPath(
...
 ): string {
     const templateFolderName = buildingBlockData.buildingBlockType;
     const templateFilePath = getTemplatePath(`/building-block/${templateFolderName}/View.xml`);
     if (typeof buildingBlockData.metaPath === 'object' || buildingBlockData.metaPath === undefined) {
+        // Special handling for chart - while runtime does not support approach without contextPath
+        // or for equal or below UI5 v1.96.0 contextPath is applied
+        const minUI5Version = manifest ? coerce(getMinimumUI5Version(manifest)) : undefined;
+        const applyContextPath =
+            buildingBlockData.buildingBlockType === BuildingBlockType.Chart ||
+            !minUI5Version ||
```
Comment (815are) [inline packages/fe-fpm-writer/src/building-block/index.ts:195]: I think if `minUI5Version` is not resolved we consider it as latest, example of existing check -> https://github.com/SAP/open-ux-tools/blob/main/packages/fe-fpm-writer/src/common/validate.ts#L16

File: packages/fe-fpm-writer/test/unit/prompts/prompts.test.ts (hunk 1 above)
Comment (AlinaGovoruhina) [reply packages/fe-fpm-writer/test/unit/prompts/prompts.test.ts]: Added cases 1.96.0 - context path applied;
1.96.25 - no context path for filter or table building blocks

File: packages/fe-fpm-writer/src/building-block/index.ts (hunk 2 above)
Comment (AlinaGovoruhina) [reply packages/fe-fpm-writer/src/building-block/index.ts]: Fixed: if not defined, contextPath is NOT applied to the building block

File: packages/fe-fpm-writer/test/unit/prompts/__snapshots__/prompts.test.ts.snap [hunk 3]
```diff
@@ -1211,6 +1230,24 @@ exports[`Prompts getCodeSnippet Type "filter-bar", get code snippet, min ui5Vers
 />"
 `;
 
+exports[`Prompts getCodeSnippet Type "filter-bar", get code snippet, min ui5Version = 1.96.25 1`] = `
+"<macros:FilterBar
+    id=\\"id\\"
+    metaPath=\\"/entity/qualifier\\"
```
Comment (815are) [inline packages/fe-fpm-writer/test/unit/prompts/__snapshots__/prompts.test.ts.snap:1236]: @AlinaGovoruhina  I think it is bug, on version `1.96.25` it should use `contextPath`. @hitesh-parmar  please correct if I am wrong

Comment (hitesh-parmar) [reply packages/fe-fpm-writer/test/unit/prompts/__snapshots__/prompts.test.ts.snap:1236]: Yes, this is correct. For all version 1.96.x there should be a contextPath

Comment (AlinaGovoruhina) [reply packages/fe-fpm-writer/test/unit/prompts/__snapshots__/prompts.test.ts.snap:1235]: This is fixed.

**Review by 815are — APPROVED** [review]
checked:
1. code is simple to understand and version check conditions fulfil expectation;
2. new tests are added
3. changeset presented

**Review by devinea — APPROVED** [review]
code changes and tests look good. 
changeset ✅

**Pull request merged**